pkg/resources/testdata/TestAcc_GrantPrivilegesToDatabaseRole_OnAllSchemasInDatabase/test.tf

```diff
@@ -0,0 +1,9 @@
+resource "snowflake_grant_privileges_to_database_role" "test" {
```

Comment:
thinking if the folder name should be 
`testdata/resources/grant_privileges_to_database_role/complete/on_all_schemas_in_database` or something to that effect. Might be better to have more levels of folders to keep things under control.

Reply:
Sure, added the main folder for all of the tests